.github/workflows/agw-build-publish-container.yml

```diff
@@ -268,4 +268,4 @@ jobs:
           SLACK_AVATAR: ":boom:"
         uses: Ilshidur/action-slack@689ad44a9c9092315abd286d0e3a9a74d31ab78a # pin@2.1.0
         with:
-          args: "Containerized LTE integration test failed on [${{ github.sha }}](${{github.event.repository.owner.html_url}}/magma/commit/${{ github.sha }}): ${{ steps.commit.outputs.title}}"
+          args: "Containerized LTE integration test failed on [${{ github.sha }}](${{github.event.repository.owner.html_url}}/magma/commit/${{ github.sha }}): ${{ github.event.head_commit.message || github.event.pull_request.title }}"
```

Comment:
In what cases would `github.event.head_commit.message` be empty so that the expression evaluates to `github.event.pull_request.title`?

Reply:
Hm, good question. I adapted the formatting of the Slack messages from the [FEG integ tests](https://github.com/magma/magma/blob/93d6eb60977ffb3205f12a171543c3f5c75e2ab0/.github/workflows/federated-integ-test.yml#L218). Looking at Slack, I would actually prefer to send the PR title. The commit message does include all commits, which can lead to a rather long message as for example [here](https://magmacore.slack.com/archives/C01QKHUEP8V/p1671128458714459). What would be your opinion?